src/main/java/de/chojo/repbot/service/ReputationService.java

```diff
@@ -10,6 +10,7 @@
 import de.chojo.repbot.data.ReputationData;
 import de.chojo.repbot.data.wrapper.GuildSettings;
 import de.chojo.repbot.util.EmojiDebug;
+import de.chojo.repbot.util.Messages;
```

Comment:
Why not use the static import below for all calls? Is there a reason, why you call them via class?

Reply:
I like this more. Static method imports mostly hide where this method comes from when reading it without and IDE which marks static imported methods

Reply:
*ohno*